tasks/add_new_openwrap_partner.py

```diff
@@ -374,13 +387,55 @@ def get_dfp_targeting(self):
             if self.bid_price: 
                 top_set['children'].append(self.bid_price)
 
+            # For HVLI, set both pwtbb and pwtecp with OR condition
+            # Targeting keys will be formed like
+            # pwtecp IS (1.*) AND pwtbst=1 AND pwtpid = abc
+            # or
+            # pwtpb = 1.0 AND pwtbst=1 AND pwtpid = abc
+            if self.price_bucket:
```

Comment:
@Pubmatic-Dhruv-Sonone and @pm-isha-bharti  :  I hope it not impacts OTT key-value targeting setup